tests/node-sync/run_full.go

```diff
@@ -80,11 +80,15 @@ func RunFullNode(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
 		return err
 	}
 
-	eh, err := nd.HeaderServ.GetByHeight(ctx, uint64(6))
+	eh, err := nd.HeaderServ.GetByHeight(ctx, uint64(12))
```

Comment:
the same: why 12?

Reply:
We are just syncing until 12. Still I'll change to a runenv value